packages/preview/sheen-id/0.1.0/README.md

```diff
@@ -0,0 +1,165 @@
+# Sheen ID - Typst Book Template
+
+A flexible and feature-rich Typst template designed for creating books. It provides a structured set of functions to handle covers, title pages, tables of contents, headers, footers, and chapter styling.
+
+## Features
+
+  - **Customizable Configuration**: Easily change the theme color, page margins, and fonts.
+  - **Automated Covers**: Generate full-page color and grayscale covers from an image.
+  - **Structured Content**: Functions for creating a title page, table of contents, list of figures, and list of equations.
+  - **Dynamic Headers & Footers**: Automatically populates with the book title and chapter titles.
+  - **Styled Chapter Pages**: A dedicated function to create beautiful chapter introduction pages with an image and learning objectives.
+  - **Pre-styled Elements**: Comes with clean, pre-defined styles for figures, tables, code blocks, math equations, and lists.
+  - **Helper Utilities**: Includes a `noindent` function to easily manage paragraph indentation.
+
+-----
+
+## Quick Start
+
+To use this template, import the `sheen-id` package and use its functions to structure your document.
+
+**`main.typ`**
+
+```typst
+// 1. Import all functions from the sheen-id package
+#import "@preview/sheen-id:0.1.0": *
+
+// 2. Apply the main configuration
+#show: doc => conf(
+  color: green, // Set the theme color
+  doc
+)
+
+// 3. Set global images (optional)
+#set-chapter-image("path/to/chapter-default.png")
```

Comment:
Setting the paths here won't work properly (even with the use of `context`), because paths are package scoped. Instead a common approach is to accept the content here directly, so users would have to pass `image("<path>")` instead. This also makes the template more flexible, since any kind of content can be used.